infinispan-remote/pom.xml

```diff
@@ -181,6 +181,15 @@
     </dependencies>
 
     <build>
+        <testResources>
+            <testResource>
+                <directory>src/test/resources</directory>
+            </testResource>
+            <testResource>
+                <directory>src/test/resources/storedprocedures/servertask</directory>
+                <targetPath>${infinispan-server.home}/standalone/deployments/</targetPath>
```

Comment:
@DavideD , and then we copy the jars from test resources to deployment directory

Reply:
great idea, let's me try it

Reply:
OK, Go for it